mteb/tasks/Clustering/eng/ArXivHierarchicalClustering.py

```diff
@@ -40,7 +40,15 @@ class ArXivHierarchicalClusteringP2P(AbsTaskClusteringFast):
         bibtex_citation="@misc{arXiv.org e-Print archive, url={https://arxiv.org/} }",
         descriptive_stats={
             "n_samples": {"test": N_SAMPLES},
-            "avg_character_length": {"test": 1009.98},
+            "avg_character_length": {
+                "test": {
+                    "num_texts": 2048,
+                    "num_labels": 2048,
```

Comment:
Yeah, num_labels is a bit meaningless since it just counts the total number of label rows. I also don't think the number of levels is useful because not all clustering tasks are hierarchical.

Reply:
But [mteb/arxiv-clustering-p2p](https://huggingface.co/datasets/mteb/arxiv-clustering-p2p) only has a list of labels, so I'm not sure how to determine if the dataset is hierarchical or not

Reply:
I believe the labels are a list:

{"text": "sample text", "label": [1, 2, 5]} # e.g. news, sports, football
{"text": "sample text", "label": [1, 2, 3]} # news sports basket

Reply:
Yes, but number of levels should be for all datasets?

Reply:
No just for clustering and potentially also multilevelclassification